docs/pages/includes/plugins/impersonations.mdx

```diff
@@ -0,0 +1,26 @@
+<Details opened={true} 
+  scopeOnly={true}
+  scope={["cloud"]}
+>
+Teleport Cloud requires authenticating with a role that has [`impersonation`](https://goteleport.com/docs/access-controls/guides/impersonation/) rights and can create the `access-plugin` role and user. Log in with `tsh` with a user that has this role or has a role with these `allow` rules.
```

Comment:
Can we replace the "impersonation" link with a one-sentence explanation of what impersonation means here? Otherwise, a reader has to leave the page and possibly break focus to complete the guide.

After the code snippet, we can say something like, "Learn more about impersonation in [Impersonating Teleport Users](path)." 

Also, we should use the "../relative/path/to/link.mdx" format so we have more control over our docs links.